handle_rentals.rb

```diff
@@ -0,0 +1,48 @@
+require './rental'
+
+class HandleRentals
+  def initialize(people, books)
+    @rentals = []
+    @handle_people = people
+    @handle_books = books
+  end
+
+  def create_rental
+    puts "Select book from this books using the number: \n"
+    @handle_books.list_of_books
+    book_number = gets.chomp
+    @handle_books.books.each_with_index { |book, index| @selected_book = book if (index + 1) == book_number.to_i }
+    puts "Select a person from the following list of members by id:\n"
+    @handle_people.list_all_people
+    person_id = gets.chomp
+    @handle_people.people.each { |person| @selected_person = person if person.id == person_id.to_i }
+    puts 'Date:'
+    date = gets.chomp
+    if @selected_person && @selected_book
+      @rentals << Rental.new(date, @selected_person, @selected_book)
+      puts 'Rental created seccussfully'
+    else
+      puts 'Something went wrong. Please try again.'
+      create_rental
+    end
+  end
+
+  def list_rentals_by_id
+    if @handle_people.people.empty?
+      puts 'There is no rentals'
+    else
+      puts 'Id:'
+      id = gets.chomp
+      @handle_people.people.each { |person| @selected_person = person if person.id == id }
+      if @selected_person == 'null'
+        puts "The member with id: #{id} doesn't have rentals"
+      else
+        puts "List of all rentals for #{@selected_person.name}, id: #{@selected_person.id}:"
+        @selected_person.rentals.each do |rental|
+          puts "- Date: #{rental.date}, Book: #{rental.book.title} by #{rental.book.author}\n"
+        end
+      end
+    end
+    @selected_person = 'null'
+  end
```

Comment:
- [ ] good job on changing the typo's notified by the previous CR 👍 however the issue when listing the rentals for a specific ID is not yet fixed, when looking for rentals from person ID 566 it provides me with the rentals of person ID 521, and when asking for the rentals for person ID 521 it says this person has no rentals
![Przechwytywanie](https://user-images.githubusercontent.com/39240820/141949173-24bb018c-4807-4ecf-bc1c-29e2350815dc.PNG)